### ISSUE TITLE
Problem with login emails

### ISSUE THREAD

**Issue by boikamlott** [trydis/FIFA-Ultimate-Team-Toolkit#224]
Hi,

I use the toolkit since 3 months now and it gave me the chance to won a lot of credits, thanks !
Since last week-end I'm in trouble with the login procedure though.
Unfortunately, I do not receive EA email login anymore (sometimes I had been temporary ban for about 12 hours...).
Any people here had this issue?
I'll try to implement the cookie workaround but just wondering if someone had the same problem recently...

I'm french BTW, sorry for my english :)

**Comment by DarkVador101** [comment]
i bet you are one of those annoying people that I come across who bot at super speeds. if that is you, then there is no wonder you are getting a 12 hour ban. if you bot smart, the most you will get is a few minutes to an hour ban

**Comment by boikamlott** [comment]
Well, actually I bot during 7-8h per day and sent 1 search request every 4-7 seconds (it's a random number between these two bounds).
Usually I'm not ban, sometimes it could append though.
But I do not think the email problem is so much related to the ban because yesterday I was fully able to log in via Firefox but no two factor email was sent whether I tried to log in from my bot or for a incognito web navigator.
Anyway I used the 2.2 version to keep the cookie in disk and use it to log in so it solved my problem.
I was just wondering if EA servers were in trouble since a couple of days or if anyone here already got the issue before...
